### PR TITLE
WF-65781-Added UG changes to find item in Word document in Java UG page

### PR DESCRIPTION
**Pull request by elizabethatienoo**
<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-65781">https://syncfusion.atlassian.net/browse/WF-65781</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Add Java UG document for find an item in Word document
</td>
</tr>
<td>
Content review confirmation
</td>
<td>
 Pending
</td>
</tr>
</table>

### CODE REVIEW

File: java-file-formats/word-library/find-item-in-word-document.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+title: Find item in Word document in Java | Syncfusion
+description: Find an item in the Word document in Java using Syncfusion Java Word library (Essential DocIO)
+platform: java-file-formats
+control: Word Library
+documentation: UG
+---
+
+# Find item in Word document
+
+Just like you can search for a text in a Word document, you can also search for an item (like image, content control, textbox, and so on). The .NET Word (DocIO) library supports to find an item in Word document based on its properties. With this functionality, you can:
```
Comment (MohanaselvamJothi) [inline java-file-formats/word-library/find-item-in-word-document.md:11]: The Java Word library supports to find a..

File: java-file-formats/word-library/find-item-in-word-document.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+* Find the first item based on one property.
+* Find the first item based on multiple properties
+* Find all the items based on one property.
+* Find all the items based on multiple properties.
+
+## Find item by property
+
+Using FindItemByProperty API, find the first item in the Word document that has the specified property name and value.
```
Comment (MohanaselvamJothi) [inline java-file-formats/word-library/find-item-in-word-document.md:20]: In Java API naming standard are different, so please use as per

Using findItemByProperty API,....

File: java-file-formats/word-library/find-item-in-word-document.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+document.save("Sample.docx",FormatType.Docx);
+//Close the Word document.
+document.close();
+{% endhighlight %}
+
+{% endtabs %}
+
+## Find item by properties
+
+Using FindItemByProperties API, find the first item in the Word document based on multiple property names and their corresponding values.
```
Comment (MohanaselvamJothi) [inline java-file-formats/word-library/find-item-in-word-document.md:47]: Using findItemByProperties API, ....

File: java-file-formats/word-library/find-item-in-word-document.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+document.save("Sample.docx",FormatType.Docx);
+//Close the Word document.
+document.close();
+{% endhighlight %}
+
+{% endtabs %}
+
+## Find all items by property
+
+Using FindAllItemsByProperty API, find all the items in the Word document that has the specified property name and value.
```
Comment (MohanaselvamJothi) [inline java-file-formats/word-library/find-item-in-word-document.md:73]: Using findAllItemsByProperty API,...

File: java-file-formats/word-library/find-item-in-word-document.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+document.save("Sample.docx",FormatType.Docx);
+//Close the Word document.
+document.close();
+{% endhighlight %}
+
+{% endtabs %}
+
+## Find all items by properties
+
+Using FindAllItemsByProperties API, find all the items in the Word document based on multiple property names and their corresponding values.
```
Comment (MohanaselvamJothi) [inline java-file-formats/word-library/find-item-in-word-document.md:111]: Using findAllItemsByProperties API,..

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: java-file-formats/word-library/find-item-in-word-document.md (hunk 5 above)
Comment (elizabethatienoo) [reply java-file-formats/word-library/find-item-in-word-document.md]: Addressed

File: java-file-formats/word-library/find-item-in-word-document.md (hunk 4 above)
Comment (elizabethatienoo) [reply java-file-formats/word-library/find-item-in-word-document.md]: Addressed

File: java-file-formats/word-library/find-item-in-word-document.md (hunk 3 above)
Comment (elizabethatienoo) [reply java-file-formats/word-library/find-item-in-word-document.md]: Addressed

File: java-file-formats/word-library/find-item-in-word-document.md (hunk 2 above)
Comment (elizabethatienoo) [reply java-file-formats/word-library/find-item-in-word-document.md]: Adressed

File: java-file-formats/word-library/find-item-in-word-document.md (hunk 1 above)
Comment (elizabethatienoo) [reply java-file-formats/word-library/find-item-in-word-document.md]: Adressed

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**